plugin/info.xml

```diff
@@ -4,7 +4,7 @@
 	<type>plugin</type>
 
 	<name>AirConsole</name>
-	<version>1.3</version>
+	<version>1.5.0.3</version>

```

Comment:
Aha - actually it was always a coincidence that it had the same version as the airconsole api. So basically we can define it as 1.4

Looking forward to your next PR! 👍 

If you want drop me a line at francois@n-dream.com with your size and address and we will send you an AirConsole tshirt. We also have an AirConsole Ambassador program if you are interested: http://developers.airconsole.com/#!/projects/ambassador